rosplan_planning_system/src/PlanParsing/PDDLEsterelPlanParser.cpp

```diff
@@ -22,6 +26,10 @@ namespace KCL_rosplan {
 		get_propositions_client = nh.serviceClient<rosplan_knowledge_msgs::GetAttributeService>(ss.str().c_str());
 		ss.str("");
 
+		ss << "/" << kb << "/state/timed_knowledge";
+		get_tils_client = nh.serviceClient<rosplan_knowledge_msgs::GetAttributeService>(ss.str().c_str());
```

Comment:
get_tils_client is also not in the current header - I guess you have an updated version.